integrationTest/test/grunt-stryker-test/stryker.conf.js

```diff
@@ -1,8 +1,8 @@
 module.exports = function (config) {
   config.set({
     files: [
-      {pattern: 'testResources/sampleProject/src/**/!(InfiniteAdd).js', mutated: true, included: true },
```

Comment:
Did the grunt integration test just never work? And we never saw it because grunt didn't care about exit code?

Reply:
There are 2 grunt integration tests, one with the config file and one without. The one that uses this config file never ran any tests, which I don't believe was intentional. It didn't fail, just ended without running any tests since the path was wrong. I fixed the path since I was using a version of this to test my change- it was really unrelated to my change.

I was initially going to add an integration test for failing the task with a break threshold, but couldn't figure out a simple way to add it and pass the build.